styles.css

```diff
@@ -78,3 +87,199 @@ div a img {
   height: 3px;
   background-color: #6070ff;
 }
+
+.project-section {
+  padding: 5% 5%;
+  gap: 50px;
+  background-color: #e5e5e5;
+}
```

Comment:
- [ ] as mentioned in the requirements, this section should be aligned with `display:grid` , kindly add that